Files/Filesystem/DriveItem.cs

```diff
@@ -8,31 +8,36 @@ namespace Files.Filesystem
     public class DriveItem : INavigationControlItem
     {
         public string Glyph { get; set; }
+        public string Text { get; set; }
+        public string Path => Tag;
+        public NavigationControlItemType ItemType { get; set; } = NavigationControlItemType.Drive;
         public ulong MaxSpace { get; set; } = 0;
         public ulong SpaceUsed { get; set; } = 0;
-        public string DriveText { get; set; }
-        public string Tag { get; set; }
```

Comment:
@duke7553 Don't we need the tag for translations purposes?

Reply:
@yaichenbaum look at line 17